[0m14:31:04.343075 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217710396A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021773A41400>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021773A41220>]}


============================== 14:31:04.358695 | 02732185-b559-43e9-ab2c-eb60d6af1e64 ==============================
[0m14:31:04.358695 [info ] [MainThread]: Running with dbt=1.8.2
[0m14:31:04.358695 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': 'C:\\Users\\looka\\.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m14:31:18.270771 [debug] [MainThread]: Starter project path: C:\ProgramData\Miniconda3\lib\site-packages\dbt\include\starter_project
[0m14:31:18.318213 [info ] [MainThread]: 
Your new dbt project "custom_postgres" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m14:31:18.319237 [info ] [MainThread]: Setting up your profile.
[0m14:32:11.142062 [error] [MainThread]: Encountered an error:

[0m14:32:11.156580 [error] [MainThread]: Traceback (most recent call last):
  File "C:\ProgramData\Miniconda3\lib\site-packages\dbt\cli\requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
  File "C:\ProgramData\Miniconda3\lib\site-packages\dbt\cli\requires.py", line 101, in wrapper
    return func(*args, **kwargs)
  File "C:\ProgramData\Miniconda3\lib\site-packages\dbt\cli\main.py", line 470, in init
    results = task.run()
  File "C:\ProgramData\Miniconda3\lib\site-packages\dbt\task\init.py", line 347, in run
    self.setup_profile(profile_name)
  File "C:\ProgramData\Miniconda3\lib\site-packages\dbt\task\init.py", line 263, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "C:\ProgramData\Miniconda3\lib\site-packages\dbt\task\init.py", line 180, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "C:\ProgramData\Miniconda3\lib\site-packages\dbt\task\init.py", line 164, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
  File "C:\ProgramData\Miniconda3\lib\site-packages\dbt\task\init.py", line 130, in generate_target_from_input
    target[key] = click.prompt(
  File "C:\ProgramData\Miniconda3\lib\site-packages\click\termui.py", line 164, in prompt
    value = prompt_func(prompt)
  File "C:\ProgramData\Miniconda3\lib\site-packages\click\termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m14:32:11.158883 [debug] [MainThread]: Command `dbt init` failed at 14:32:11.158883 after 66.84 seconds
[0m14:32:11.159487 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217710396A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021773B014C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000217735F5580>]}
[0m14:32:11.160605 [debug] [MainThread]: Flushing usage events
